racingcar-java/src/main/java/mission02/Calculator.java

```diff
@@ -0,0 +1,26 @@
+package mission02;
+
+/**
+ * 최종적인 계산기, main에 우겨 때려 넣을 수 없어서 팩토리 하나를 만듦
+ */
+public class Calculator {
+
+    private final StringUtils stringUtils;
+    private final Calculation calculation;
+
+    public Calculator(String expression) {
+        stringUtils = new StringUtils(expression);
+        calculation = new Calculation();
+    }
+
+    public int getCalculatorValue(){
+        String[] expressToken = stringUtils.getExpression();
+        int base = stringUtils.getCharacterToNumber(expressToken[0]);
```

Comment:
```suggestion
        String[] expressToken = this.expression.split(" ");
        int base = StringUtils.characterToNumber(expressToken[0]);
```
말한대로 StringUtils 객체를 만들어 이를 처리하는게 아니라
expression필드를 저장하고 StringUtils는 단지 필드를 받아 처리해주는 util적인 역할만 해주어야할 것 같네요